tests/acceptance/test_data/templates/expected_plugin_help_output.txt

```diff
@@ -30,10 +30,10 @@ Flags:
   -h, --help   Help for plugin
 
 Global Flags:
-      --cloud-host string           Steampipe Cloud host (default "cloud.steampipe.io")
-      --cloud-token string          Steampipe Cloud authentication token
+      --cloud-host string           Turbot Pipes host (default "pipes.turbot.com")
```

Comment:
it was agreed we leave the arg names as `cloud` ?

Reply:
Yes. Only the defaults should change.

Quoting @johnsmyth :
> The default value for the cloud host (STEAMPIPE_CLOUD_HOST/cloud_host/--cloud-host) is `[cloud.steampipe.io](http://cloud.steampipe.io/)`.  It should be changed to `[pipes.turbot.com](http://pipes.turbot.com/)`